project2/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Catamaran:900&display=swap" rel="stylesheet">
+    <title>Movie Cards</title>
+</head>
+
+<body>
+    <section class="card">
+        <h2 class="title">joker</h2>
+        <div class="score"><span class="imbd">9.4</span>/10</div>
+        <ul class="rating-time-genres">
+            <li>R</li>
+            <li>2h 1min</li>
+            <li>Crime, Drama, Thriller</li>
+        </ul>
+        <a class="button" href="https://www.imdb.com/title/tt7286456/videoplayer/vi1723318041?ref_=tt_ov_vi">watch
+            trailer</a>
+        </div>
+        <div class="gradient"></div>
+        <div class="button-shadow"></div>
+        <img class="poster" src="images/jokerposter.jpg" alt="a poster of joker">
+    </section>
```

Comment:
You're missing the closing `body` and `html` tags here:
```suggestion
    </section>
</body>

</html>    
```